ovirtlago/cmd.py

```diff
@@ -157,6 +162,7 @@ def do_ovirt_reposetup(
     if kwargs['ioprocess_dir']:
         warnings.warn('Deprecated option --ioprocess-dir ignored')
 
+    _repo_path = kwargs['repo_path']
```

Comment:
as this is a non-deprecated option, is better if you declare it as a parameter of the function (just as prefix or rpm_repo), that way you don't need to access the kwargs yourself